formats/shared/definitions/_specialist_document.jsonnet

```diff
@@ -1800,6 +1803,60 @@
       },
     },
   },
+  marine_notice_metadata: {
+    type: "object",
+    additionalProperties: false,
+    properties: {
+      bulk_published: {
+        type: "boolean",
+      },
+      marine_notice_type: {
+        type: "string",
+        enum: [
+          "marine-guidance-note",
+          "merchant-shipping-notice",
+          "marine-information-note",
+          "form",
+          "instructions-to-surveyors",
+          "codes-of-practice",
+          "safety-bulletin",
+          "other",
+        ],
+      },
+      marine_notice_vessel_type: {
+        type: "string",
+        enum: [
+          "pleasure-vessels",
+          "small-commercial-vessels",
+          "cargo-vessels-over-24-metres",
+          "fishing-vessels-under-15-metres",
+          "fishing-vessels-15-24-metres",
+          "fishing-vessels-over-24-metres",
+          "high-speed-craft",
+          "large-yachts",
+          "passenger-vessels",
+        ],
+      },
+      marine_notice_topic: {
+        type: "string",
+        enum: [
+          "construction-and-equipment",
+          "crew-and-training",
+          "health-and-safety",
+          "environmental",
+          "navigation",
+          "radio-communications",
+          "registration",
+          "security-isps",
+          "survey-and-inspection",
+        ],
+      },
+      marine_notice_issued_date: {
```

Comment:
Should this be `marine_notice_issued_date` or just `issued_date`?  The `medical_safety_alert_metadata` type uses `issued_date`

Reply:
it needs to match whatever field is declared in specialist publisher, but yeah, it would indeed be best to reuse fields whenever possible 👌